tinymce.builder.inc

```diff
@@ -708,6 +708,11 @@ function _tinymce_get_button_defs() {
       'icon' => 'paste-text',
       'tooltip' => t('Paste as text'),
     ),
+    'anchor' => array(
```

Comment:
Nit pick, but this is the wrong section. :wink: 

Could you move it above, right after accordion, and add the tags info as `'required_tags' => array('a'),`?
Will try to test soon.

Reply:
Never mind, it turned out, the backdroplink plugin needed a tiny adaption, anyway to not compete with the anchor plugin.